### ISSUE TITLE
New Version

### ISSUE THREAD

**Issue by fxfactorial** [obihann/archey-osx#19]
Hi,

I think I just got a new version of this via `brew` and Its actually annoying that you are hitting `dig` each time. This makes opening a new shell takes very much longer than if one has a call to `archey` in their respective `.zshrc/.bashrc` Can you cache this somehow, I like the feature just not the new call each time.

**Comment by obihann** [comment]
Hi @fxfactorial,

I haven't released a new version in a while, I was about to however some issues came up and I wanted to look into them first. One is related to dig so I will hopefully be finding a solution to this.

Thanks.

**Comment by fxfactorial** [comment]
Free suggestion?

make a file with the IP address, then on each usage of archey do a stat on it, if time difference from now is greater than say six hours then refresh with new call to dig, otherwise use the current value.

**Comment by obihann** [comment]
Could you run `$ brew info archey` for me and let me know the results? The colour I know was introduced a while ago but it (from what I remember) wasn't a default setting, it required `$ archey -c` to turn it on...

**Comment by fxfactorial** [comment]
```shell
~ ❯❯❯ brew info archey
archey: stable 1.5, HEAD
Archey script for OS X
https://obihann.github.io/archey-osx/
/usr/local/Cellar/archey/1.5 (4 files, 16K) *
  Built from source
From: https://github.com/Homebrew/homebrew/blob/master/Library/Formula/archey.rb
```

**Comment by obihann** [comment]
@fxfactorial Somebody else released version 1.5 for me and I didn't even notice, so that explains why you noticed the change. I'll look into this soon and get back to you. Thanks!
